### PR TITLE
Invoice item info

### PR DESCRIPTION
**Pull request by kenzjoy**
- added an instance method for invoice so we have access to invoice_items data
- test for edge case that merchant only sees items from their shop on the invoice 
- 100% coverage on all tests

### CODE REVIEW

File: app/models/invoice.rb [hunk 1]
```diff
@@ -6,4 +6,8 @@ class Invoice < ApplicationRecord
   has_one :merchant, through: :items
 
   enum status: [ :"in progress", :cancelled, :completed ]
+
+  def invoice_item(item_id)
+    invoice_items.find_by(item_id: item_id)
```
Comment (msakr21) [inline app/models/invoice.rb:11]: I like the idea of setting this up for use in the iteration in the show page!

**Review by msakr21 — APPROVED** [review]
Good job! Going to go ahead and merge!
